libmscore/mscore.cpp

```diff
@@ -338,6 +346,41 @@ void MScore::init()
                   }
             }
 
+      // 4 types of paper sizes: Common, Metric, Imperial, Other
+      // Common overlaps with Metric/Imperial/Other and is sorted manually here
+      // Envelope sizes excluded
+      unsigned i;
+      sizesCommon.push_back(QPageSize::A3);
+      sizesCommon.push_back(QPageSize::A4);
+      sizesCommon.push_back(QPageSize::A5);
+      sizesCommon.push_back(QPageSize::B5);
+      sizesCommon.push_back(QPageSize::JisB4);
+      sizesCommon.push_back(QPageSize::Letter);
+      sizesCommon.push_back(QPageSize::Legal);
+      sizesCommon.push_back(QPageSize::Ledger);
+      sizesCommon.push_back(QPageSize::Tabloid);
+      sizesCommon.push_back(QPageSize::Imperial9x12);
+      sizesCommon.push_back(QPageSize::Imperial10x13);
```

Comment:
It would probably be better to initialize it with initializer list rather than pushing each value separately. It would be both faster and more readable.

Reply:
I don't know how to do that with `std::vector` and `std::set`.  I looked online for help or examples and found none.  If you can show me how, I'll do it.

Reply:
See for example https://github.com/musescore/MuseScore/blob/master/effects/compressor/compressor.cpp#L36

Reply:
Do you want me to change this for the `std::set` variables too, or just for the vector with the odd sort order, `sizesCommon`?  The for loops are convenient for the other 3, but I can initialize them with literal values too, if that is preferred.  This is fixed in the next push.

Reply:
In the next push you'll see a new `MScore::initPageSizes()` function.  Along the same lines, it's tempting to create a `MScore::initStyles()` function.